code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1750,4 +1750,21 @@
 
 	hud_set_squad()
 
-	return TRUE
\ No newline at end of file
+	return TRUE
+
+/mob/living/carbon/human/screech_act(mob/living/carbon/Xenomorph/Queen/Q)
+	shake_camera(src, 3 SECONDS, 1) //50 deciseconds, SORRY 5 seconds was way too long. 3 seconds now
```

Comment:
The prior behavior checked for client before shaking camera:
```
if(client)
	shake_camera(src, 3 SECONDS, 1) //50 deciseconds, SORRY 5 seconds was way too long. 3 seconds now

Reply:
`shake_camera()` checks for client which is why i removed it.